README.md

```diff
@@ -57,11 +57,15 @@ After running `docker-compose up` successfully, the output in your terminal shou
 
     > Note: Comes with a full PostgreSQL installation.
 
-- [GDAL (Geospatial Data Abstraction Library)](GDAL_LIBRARY_PATH) (GIS support for PostgreSQL)
+- GDAL (Geospatial Data Abstraction Library) (GIS support for PostgreSQL)
 
     > Note: You may need to set the GDAL_LIBRARY_PATH and/or GEOS_LIBRARY_PATH
-    > environment variables in your shell environment.
+    > environment variables in your shell environment. Remember to add export in front of the variable.
 
+- [ImageMagick] (https://formulae.brew.sh/formula/imagemagick)
```

Comment:
nit: I think the formatting here is off

<img width="990" alt="Screenshot 2024-04-03 at 5 55 33 PM" src="https://github.com/CodeWithAloha/uipa/assets/15609358/155a087c-d848-4602-bb26-d1f4851518cb">